Plugins/NativeInput/Include/Babylon/Plugins/NativeInput.h

```diff
@@ -12,9 +12,12 @@ namespace Babylon::Plugins
         static NativeInput& CreateForJavaScript(Napi::Env);
         static NativeInput& GetFromJavaScript(Napi::Env);
 
-        void PointerDown(uint32_t pointerId, uint32_t buttonIndex, uint32_t x, uint32_t y);
-        void PointerUp(uint32_t pointerId, uint32_t buttonIndex, uint32_t x, uint32_t y);
-        void PointerMove(uint32_t pointerId, uint32_t x, uint32_t y);
+        void MouseDown(uint32_t buttonIndex, uint32_t x, uint32_t y);
+        void MouseUp(uint32_t buttonIndex, uint32_t x, uint32_t y);
+        void MouseMove(uint32_t x, uint32_t y);
+        void TouchDown(uint32_t pointerId, uint32_t buttonIndex, uint32_t x, uint32_t y);
+        void TouchUp(uint32_t pointerId, uint32_t buttonIndex, uint32_t x, uint32_t y);
```

Comment:
I don't think we need/want `buttonIndex` for these since there is only one "button" for touch. Seems like it should be:
```suggestion
        void TouchDown(uint32_t pointerId, uint32_t x, uint32_t y);
        void TouchUp(uint32_t pointerId, uint32_t x, uint32_t y);
```

Reply:
sounds good